.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a bug report to help improve depstat dependency analyzer CLI
+title: ''
+labels: 'kind/bug'
+assignees: ''
+
+---
+
+<!-- Please answer these questions before submitting your bug report. Thanks! -->
+
+**What version of depstat are you using?**
+
+depstat version:
+
+**Does this issue reproduce with the latest release?**
+
+**Which depstat CLI options are you using?**
```

Comment:
I think we can remove this since this will be answered in "What did you do?" anyways.